openedx/core/djangoapps/course_groups/cohorts.py

```diff
@@ -189,27 +194,37 @@ def get_cohort(user, course_key, assign=True):
     Raises:
        ValueError if the CourseKey doesn't exist.
     """
+    request_cache = RequestCache.get_request_cache()
```

Comment:
@cpennington: Just want to double check with you that this is the right way to use a request cache.